contracts/mocks/WhitelistedCrowdsaleImpl.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.18;
+
+//import "../examples/SimpleToken.sol";
+import "../token/ERC20/ERC20.sol";
+import "../crowdsale-refactor/validation/WhitelistedCrowdsale.sol";
+
+
+contract WhitelistedCrowdsaleImpl is WhitelistedCrowdsale {
+
+  function WhitelistedCrowdsaleImpl (
+    uint256 _rate,
+    address _wallet,
+    ERC20 _token
+  ) public
+    Crowdsale(_rate, _wallet, _token)
+    //Ownable() //Do I need to call it anyway?! --apparently not
```

Comment:
Base constructors only need to be referenced when you need to pass parameters to them, so no, Ownable is usually not needed to be referenced.
Can we remove the comment here?

Reply:
Sorry, left this behind, removed.